src/Compilers/CSharp/Portable/Symbols/FunctionPointers/FunctionPointerMethodSymbol.cs

```diff
@@ -483,6 +559,62 @@ internal FunctionPointerMethodSymbol ApplyNullableTransforms(byte defaultTransfo
             }
         }
 
+        public ImmutableHashSet<CustomModifier> GetCallingConventionModifiers()
```

Comment:
>GetCallingConventionModifiers [](start = 48, length = 29)

It feels like we want to have tests with:
- duplicate modifiers
- with unrelated modifiers before, after, and in between
- interesting modifiers before and after ref byte at the same time, those after the ref bite shouldn't affect the calling convention matching.

Reply:
We have the first two (see `Equality_UnmanagedExtensionModifiers`). I'll add the last one.

---
In reply to: [458297902](https://github.com/dotnet/roslyn/pull/46110#discussion_r458297902) [](ancestors = 458297902)

Reply:
Added `Equality_DifferingRefAndTypeCustomModifiers`

---
In reply to: [458447003](https://github.com/dotnet/roslyn/pull/46110#discussion_r458447003) [](ancestors = 458447003,458297902)